python/mlc_llm/model/llava/llava_model.py

```diff
@@ -0,0 +1,620 @@
+                        self.config.vision_config.image_size,
+                    ],
+                    self.dtype,
+                ),
+                "$": {
+                    "param_mode": "packed",
+                    "effect_mode": "none",
+                },
+            },
+            "prefill": {
+                "input_embed": nn.spec.Tensor(
+                    [1, "seq_len", self.config.text_config.hidden_size], self.dtype
+                ),
+                "paged_kv_cache": nn.spec.Object(object_type=PagedKVCache),
+                "$": {
+                    "param_mode": "packed",
+                    "effect_mode": "none",
+                },
+            },
+            "decode": {
+                "input_embed": nn.spec.Tensor(
+                    [1, 1, self.config.text_config.hidden_size], self.dtype
+                ),
+                "paged_kv_cache": nn.spec.Object(object_type=PagedKVCache),
+                "$": {
+                    "param_mode": "packed",
+                    "effect_mode": "none",
+                },
+            },
+            "batch_prefill": {
+                "input_embeds": nn.spec.Tensor(
+                    [1, "seq_len", self.config.text_config.hidden_size], self.dtype
+                ),
+                "logit_positions": nn.spec.Tensor(["batch_size"], "int32"),
+                "paged_kv_cache": nn.spec.Object(object_type=PagedKVCache),
+                "$": {
+                    "param_mode": "packed",
+                    "effect_mode": "none",
+                },
+            },
+            "batch_decode": {
+                "input_embeds": nn.spec.Tensor(
+                    ["batch_size", 1, self.config.text_config.hidden_size], self.dtype
+                ),
+                "paged_kv_cache": nn.spec.Object(object_type=PagedKVCache),
+                "$": {
+                    "param_mode": "packed",
+                    "effect_mode": "none",
+                },
+            },
+            "batch_verify": {
+                "input_embeds": nn.spec.Tensor(
+                    [1, "seq_len", self.config.text_config.hidden_size], self.dtype
+                ),
+                "paged_kv_cache": nn.spec.Object(object_type=PagedKVCache),
+                "$": {
+                    "param_mode": "packed",
+                    "effect_mode": "none",
+                },
+            },
+            "softmax_with_temperature": {
+                "logits": nn.spec.Tensor(["batch_size", 1, "vocab_size"], "float32"),
+                "temperature": nn.spec.Tensor(["batch_size"], "float32"),
+                "$": {
+                    "param_mode": "none",
+                    "effect_mode": "none",
+                },
+            },
+            "create_paged_kv_cache": {
+                "max_batch_size": int,
+                "max_total_seq_len": int,
+                "prefill_chunk_size": int,
+                "page_size": int,
```

Comment:
#1967 supports another parameter `support_sliding_window`. You may need to rebase to the latest main and add the parameter here and in the definition of `create_paged_kv_cache`.

Reply:
And also rebase to resolve the conflict in `conversation_protocol.py`.

Reply:
I did something wrong when rebasing. Closing this PR and opening a new one. Sorry for that